lib/cucumber/support_code/library.js

```diff
@@ -22,6 +22,18 @@ function Library(supportCodeDefinition) {
     });
   }
 
+  function getStackframe() {
+    var stackframes = StackTrace.getSync();
+    for (var i = 0; i < stackframes.length; i++) {
+      if (stackframes[i].fileName.indexOf('/lib/cucumber/') === -1 &&
+          stackframes[i].fileName !== '(native)') {
+        return stackframes[i];
+      }
+    }
```

Comment:
Please use lodash's `_.find` here instead of the manual for loop and `_.includes` instead of `stackframes[i].fileName.indexOf`.